tests/modules/postnotifications.c

```diff
@@ -213,7 +214,7 @@ static void KeySpace_ServerEventCallback(RedisModuleCtx *ctx, RedisModuleEvent e
     REDISMODULE_NOT_USED(eid);
     REDISMODULE_NOT_USED(data);
     if (subevent > 3) {
-        RedisModule_Log(ctx, "warning", "Got an unexpected subevent '%ld'", subevent);
+        RedisModule_Log(ctx, "warning", "Got an unexpected subevent '%" PRIu64 "'", subevent);
```

Comment:
so far we've avoided to use this notation (`PRIu64` and such).
IIRC Salvatore pushed back against it.
how about doing casting instead?

Reply:
That's what it was before I commented on it. :) See my comment above.

Btw, what's the reasoning against using these macros? They are C99.

Reply:
i don't know. maybe it's more readable (to me it is :smile:).
or maybe just not making a precedence. i don't recall where the previous discussion on that was.